addons/l10n_sa_invoice/models/account_move.py

```diff
@@ -39,11 +40,11 @@ def get_qr_encoding(tag, field):
                 company_vat_enc = get_qr_encoding(2, record.company_id.vat)
                 time_sa = fields.Datetime.context_timestamp(self.with_context(tz='Asia/Riyadh'), record.l10n_sa_confirmation_datetime)
                 timestamp_enc = get_qr_encoding(3, time_sa.isoformat())
-                invoice_total_enc = get_qr_encoding(4, str(record.amount_total))
-                total_vat_enc = get_qr_encoding(5, str(record.currency_id.round(record.amount_total - record.amount_untaxed)))
+                invoice_total_enc = get_qr_encoding(4, float_repr(abs(record.amount_total_signed), 2))
+                total_vat_enc = get_qr_encoding(5, float_repr(abs(record.amount_tax_signed), 2))
```

Comment:
The amount will be positive for inbound move types and negative for outbound types, whereas it was always positive before.
Is this wanted?

Reply:
I missed the `abs` before, my bad